modules/virt-tested-cluster-maximums.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-tested-cluster-maximums.adoc
+
+[id="virt-tested-cluster-maximums_{context}"]
+= Tested cluster maximums for {VirtProductName}
+
+The maximum number of virtual machines in a node is determined by a number of variables that include the virtual machines' CPUs, memory, and network characteristics.
+
+The following table shows the tested cluster maximums for the {VirtProductName} 4.8 release.
```

Comment:
If this module is going to be maintained to describe the current release only, I recommend using the `{VirtVersion}` attribute instead of hardcoding "4.8" into the file

Reply:
Added {VirtVersion} since the results of the tests apply only to the 4.8 release.